config/packages/framework.yaml

```diff
@@ -2,22 +2,15 @@
 framework:
     secret: '%env(APP_SECRET)%'
     #csrf_protection: true
-    annotations: true # required for rest routing loader to be true
```

Comment:
the rest routing loader not longer depence on the annotations reader of symfony instead creating its own instance if symfony annotation loader is not available.